core/src/runtime/RuntimeClient.ts

```
@@ -192,6 +192,24 @@ export class RuntimeClient {
     }
 
+    async getNetworkMetrics(): Promise<String> {
+        const { runtimeGetNetworkMetrics } = unwrapApolloResult(await this.#apolloClient.query({
+            query: gql`query runtimeGetNetworkMetrics {
+                runtimeGetNetworkMetrics
+            }`,
+        }))
+        return runtimeGetNetworkMetrics
+    }
+
+    async restartHolochain(): Promise<Boolean> {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)